flink-cdc-connect/flink-cdc-pipeline-connectors/flink-cdc-pipeline-connector-elasticsearch/src/test/java/org/apache/flink/cdc/connectors/elasticsearch/sink/Elasticsearch7DataSinkITCaseTest.java

```diff
@@ -137,7 +137,7 @@ public void testElasticsearchInsertAndDelete() throws Exception {
     @Test
     public void testElasticsearchAddColumn() throws Exception {
         TableId tableId = TableId.tableId("default", "schema", "table");
-        List<Event> events = ElasticsearchTestUtils.createTestEventsWithAddColumn(tableId); // 使用工具类
```

Comment:
could you help check all Chinese note?

Reply:
Grepped all files and found no more comments written in Chinese.